go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/go-critic/go-critic
 
+go 1.13
```

Comment:
```suggestion
go 1.12
```

Reply:
I'll use go 1.12 to re-generate those files. :)